app/root.tsx

```diff
@@ -16,9 +18,11 @@ export default function App() {
 		<Document nonce={nonce}>
 			<div className="flex h-screen flex-col justify-between">
 				<div className="flex-1">
+					<HeaderWithSearch />
 					<main className="grid h-full place-items-center">
-						<h1 className="text-mega">Welcome to Epic News!</h1>
+						<h1 className="text-mega">Your Journey Begins!</h1>
 					</main>
+					<FooterMenuRight />
```

Comment:
Can you add your logo to `FooterMenuRight`? Check the [guide](https://gedsargent.github.io/code-monster/guides/epic-news/ui/import-image-to-navbar-and-footer/#importing-the-image) on Code Monster for help importing images and using them in components.